content/blog/02_mineout.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Making Of: Mineout"
+date = 2021-01-06
+
+extra.image = "assets/images/mineout.png"
++++
+
+As described by the homepage, 
+> Mineout is a free-for-all party game in which players must reach the end of a set course. The default map has four sections:
+> * Bridging: players must bridge to the other side of a large gap while it decays behind them
+> * Elytra: players must fly through scattered blocks to get to the other side
+> * Riptide: players must use a Riptide trident to fly through hoops
+> * Parkour: players must cross a lava pit by parkour
+The game was devised by mounderfod (I'm writing this thing, hi there!) and both Marth_Daul and SillySausage09 helped with the building. Gegy then programmed the whole minigame in Plasmid.
+
+# Original Design
+The minigame is based on *To Get To The Other Side (TGTTOS)*, which is one of the games present in the popular [Minecraft Championship](https://noxcrew.com/mcc). TGTTOS involves a number of rounds in which the players work to, well, *get to the other side*, using various means of transport to reach the end goal each time. There are parkour rounds, bridging rounds, and even a trident Riptiding round. No other server that I knew of had a minigame anything like this, and at the time I was a builder for Nucleoid, and so I made the decision to open a map and try to build a similar game. 
+
+# Mineout v1
+The original Mineout which I had devised had four sections in order: parkour, elytra flight, riptiding and bridging. It took me a few days to build the map but with the help of SillySausage and Marth, I was able to get it done. Most of the parkour and elytra section was randomly generated using WorldEdit, and then modified to make it playable. There were a few issues that we had to solve, particularly with the elytra and trident sections. How could we stop the player from going in the wrong place? Eventually, we decided to use water to catch the players in the elytra section, and then they could swim down a small opening into the trident section (1). If they were flying lower, then they could pass straight through (2). <br>
+
+<img src="/static/assets/images/blog/mineout_diagram.png" width="500px">
```

Comment:
Can this cause images to potentially overflow if the screen is smaller than 500px?

Reply:
This image is also quite big- might be worth reducing the resolution

Reply:
Is `30%` better?

Reply:
Oh- not in terms of how it displays, but rather the actual file size.

Reply:
I'm not sure how to fix this?

Reply:
compress it in a different format like jpeg or such